examples/trajectory.ipynb

```diff
@@ -0,0 +1,209 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import xarray as xr\n",
+    "import datashader as ds\n",
+    "import datashader.transfer_functions as tf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create some fake data\n",
+    "\n",
+    "Here we create a fake trajectory in a 2D space by calculating a random walk with momentum and then adding various perturbations:\n",
+    "\n",
+    "1. a sine-based displacement of the x axis to simulate e.g. a mechanical problem\n",
+    "2. random noise on both x and y to simulate measurement uncertainty\n",
+    "3. a completely arbitrary bogus value at a fixed location, to simulate corrupted data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Constants\n",
+    "np.random.seed(1)\n",
+    "n = 1000000 # Number of points\n",
+    "f = filter_width = 5000 # momentum or smoothing parameter, for a moving average filter\n",
+    "\n",
+    "# filtered random walk\n",
+    "xs = np.convolve(np.random.normal(0, 0.1, size=n), np.ones(f)/f).cumsum()\n",
+    "ys = np.convolve(np.random.normal(0, 0.1, size=n), np.ones(f)/f).cumsum()\n",
+    "\n",
+    "# Add \"mechanical\" wobble on the x axis\n",
+    "xs += 0.1*np.sin(0.1*np.array(range(n-1+f)))\n",
+    "\n",
+    "# Add \"measurement\" noise\n",
+    "xs += np.random.normal(0, 0.005, size=n-1+f)\n",
+    "ys += np.random.normal(0, 0.005, size=n-1+f)\n",
+    "\n",
+    "# Add a completely incorrect value\n",
+    "xs[int(len(xs)/2)] = 100\n",
+    "ys[int(len(xs)/2)] = 0\n",
+    "\n",
+    "# Create a dataframe\n",
+    "df = pd.DataFrame(dict(x=xs,y=ys))\n",
+    "\n",
+    "# Default plot ranges:\n",
+    "x_range = (xs.min(), xs.max())\n",
+    "y_range = (ys.min(), ys.max())\n",
+    "\n",
+    "df.tail()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Static Plots\n",
+    "\n",
+    "With datashader, it's quick and easy to plot the entire 1-million-point dataset, without any downsampling:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def create_image(x_range=x_range, y_range=y_range, w=500, h=500):\n",
+    "    cvs = ds.Canvas(x_range=x_range, y_range=y_range, plot_height=int(h), plot_width=int(w))\n",
```

Comment:
At the moment those notebooks are on Anaconda Cloud, and if some one gets them from there they'll work with the current github master only if the int() calls are in there.  Once this PR is merged, then those calls will no longer be needed.  I guess I can remove them from the versions in this PR, as long as I don't later need to update Anaconda Cloud before the PR is merged...